src/main/java/engine/config/reader/ConfigReader.java

```diff
@@ -0,0 +1,27 @@
+package engine.config.reader;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigReader {
+    public static String getProperty(PropertyKeys key) {
```

Comment:
@Fishgame1 are we ok with the fact that properties file will be loaded each time single property is retrieved?

Reply:
No, but @jakub-krzysiek had an info dump so big yesterday that I didn't want to introduce the singleton as well:) I will speak with him today about it :) but thanks for the comment cause of course I forgot about that in the meanwhile :)